src/types.js

```diff
@@ -31,6 +31,110 @@ export type AuthenticationOptions = {
     cryptoKeyPath: string,
 };
 
+/**
+ * Options for configuring the keep alive behavior.
+ */
+export type KeepAliveOptions = {
+    /**
+     * Sets the working socket to timeout after workingSocketTimeout
+     * milliseconds of inactivity on the working socket.
+     * Default is freeSocketTimeout * 2.
+     */
+    workingSocketTimeout?: number,
+
+    /**
+     * Sets the free socket to timeout after freeSocketTimeout milliseconds
+     * of inactivity on the free socket. Default is 15000.
+     */
+    freeSocketTimeout?: number,
+
+    /**
+     * Maximum number of sockets to allow per host. Default is Infinity.
+     */
+    maxSockets?: number,
+
+    /**
+     * Maximum number of sockets (per host) to leave open in a free state.
+     * Default is 256.
+     */
+    maxFreeSockets?: number,
+};
+
+/**
+ * Options to configure the request caching strategy.
+ */
+export type CachingStrategy = {
+    /**
+     * The superagent-cache-plugin-compatible cache that will be used as the
+     * cache implementation.
+     */
+    provider: mixed,
+
+    /**
+     * A callback to calculate when the cached response for a given URL should
+     * expire. If this method is omitted, the cache provider's default
+     * expiration will be used. The result is given to superagent-cache-plugin
+     * and works according to its documentation.
```

Comment:
This is no longer true if we use v3

Reply:
Yeah, I know. I didn't realise it had installed the beta.